mmv1/third_party/terraform/services/bigtable/resource_bigtable_instance.go

```diff
@@ -624,11 +642,6 @@ func resourceBigtableInstanceClusterReorderTypeList(_ context.Context, diff *sch
 		}
 	}
 
-	err := diff.SetNew("cluster", orderedClusters)
```

Comment:
I'm not entirely certain that this is the cause of the failures, but I think that `diff.SetNew` modified the internal state of the resource, affecting future `diff` method calls, and we're seeing different behaviour now that's caused a handful of tests to fail. We may need to break up the function to end here, starting a new function afterwards, and combine them. (We could also stub out the behaviour of the ResourceDiff object, but I am not confident we'd accurately recreate it in a stub)

Reply:
Fixed